pandas/io/formats/printing.py

```diff
@@ -391,6 +395,52 @@ def best_len(values):
     return summary
 
 
+def _justify(head, tail, display_width, best_len,
+             is_truncated=False, is_multi=False):
+    """
+    Justify each item in head and tail, so they align properly.
+    """
+    if is_multi:
```

Comment:
this is getting pretty complicated. e.g. the nested calling of this. maybe ban justify / is_multi

Reply:
is_multi also needs to justify, but on each value in the tuple for each value, instead of a flexible list of values.

I see it's a bit complicated, but it's also difficult to make it simpler. I've tried containng the new functionality, maybe it's better.